src/docfx/build/context/Context.cs

```diff
@@ -64,6 +66,8 @@ internal sealed class Context : IDisposable
 
         public LocalizationProvider LocalizationProvider { get; }
 
+        public Validator DocsValidator { get; }
```

Comment:
Instead of having logic [here](https://github.com/dotnet/docfx/pull/5656/files#diff-d898f1c3e2ee7e44ae96aa08cccd7a49R132), [here](https://github.com/dotnet/docfx/pull/5656/files#diff-c329d0f4c7d5076b957537c2818b4a03R383) and [here](https://github.com/dotnet/docfx/pull/5656/files#diff-16e9cfd7374b164c5e3c3043be71202eR65), consider creating a wrapper class like `ContentValidator` that house the adaption logic. 
```suggestion
        public ContentValidator ContentValidator { get; }
```

Reply:
good suggestion